### DISCUSSION TITLE
What version of Process Hacker is this?

### DISCUSSION THREAD

**Discussion by Aholicknight** [winsiderss/systeminformer#1135] [Questions / Answers] [answered]
![image](https://user-images.githubusercontent.com/7843719/155039200-8cf0f3ad-4642-46e8-be3e-48c10eb47441.png)

Sorry for asking such a stupid question, but what version of Process Hacker is this? This is included with a reverse engineering pack and was blown away by all its features from the process hacker I use. 
![image](https://user-images.githubusercontent.com/7843719/155039015-cb189d48-f027-47c1-8503-25bdcd11c931.png)
![image](https://user-images.githubusercontent.com/7843719/155039149-c8922e63-e64a-4549-832a-6412a94ec822.png)
It has way more choices when you right-click an exe (Freeze, etc.) + way more options

Thank you!

**Comment by yfdyh000** [comment, marked as answer]
See the https://processhacker.sourceforge.io/nightly.php.

**Reply by Aholicknight** [reply]
> See the [processhacker.sourceforge.io/nightly.php](https://processhacker.sourceforge.io/nightly.php).

This is the nightly build?

**Reply by yfdyh000** [reply]
3.x in nightly branch.

**Reply by Aholicknight** [reply]
> 3.x in nightly branch.

Well I guess that solves my question, thank you!
